contracts/starknet/dai_join.cairo

```diff
@@ -16,10 +16,10 @@
 %lang starknet
 
 from starkware.cairo.common.cairo_builtins import HashBuiltin
-from starkware.starknet.common.syscalls import (get_contract_address, get_caller_address)
-from starkware.cairo.common.uint256 import (Uint256, split_64)
-from contracts.safe_math import (mul)
-from contracts.assertions import (check)
+from starkware.starknet.common.syscalls import get_contract_address, get_caller_address
+from starkware.cairo.common.uint256 import Uint256, split_64
+from contracts.starknet.safe_math import mul
```

Comment:
`contracts.starknet` seems very adhoc. It should be either local import like `from safe_math` or  `from makerdao.starknet.domain.safe_math`. It is a general comment, we should think about it, does not need to be implemented now.